### PR TITLE
Add birth date to adoption app (#1185)

### PR DESCRIPTION
**Pull request by markottaviani**
* Add birth date and adopter age to adoption app
* Add birth date and adopter age to adopt app mailer and add mailer preview
* Fix adopter age in adoption app and add tests
* Fix comment in mailer preview and make adoption age plural in all cases except for 1

(pulled into repo so that TravisCI can run)

### CODE REVIEW

File: spec/models/adoption_app_spec.rb [hunk 1]
```diff
@@ -56,4 +57,36 @@
...
+      expect(adoption_app.adopter_age).to eq("10 years")
+    end
+
+    it 'returns 9 years when birth_date is 1 month before 10 years from current date' do
+      adoption_app.birth_date = (10.years.ago + 1.month).to_date
+
+      expect(adoption_app.adopter_age).to eq("9 years")
+    end
+
+    it 'returns 9 years when birth_date is 1 day before 10 years from current date' do
```
Comment (markottaviani) [inline spec/models/adoption_app_spec.rb]: @nickcianci turns out this test will fail on late night CI runs :)    If UTC date and ET date are different this test will fail.  Given the org is fully Eastern Time based, it's probably best to honor local timezone of the app for determining age.  

I've added a quick time helper to consistently break the test which should make debugging a bit easier.  You should be able to push / pull directly from this feature branch.

Comment (nickcianci) [reply spec/models/adoption_app_spec.rb]: I'm not able to replicate. Isn't it already honoring local timezone? I'm not honoring local timezone by using `10.years.ago + 1.day`?

Comment (markottaviani) [reply spec/models/adoption_app_spec.rb]: Not sure, are you able to see https://travis-ci.org/ophrescue/RescueRails/jobs/647725489#L1778  ?

Comment (nickcianci) [reply spec/models/adoption_app_spec.rb]: yes, can we print out the current datetime from the build server when running the test?

Comment (markottaviani) [reply spec/models/adoption_app_spec.rb]: Sure can, just pushed some `puts`

Comment (nickcianci) [reply spec/models/adoption_app_spec.rb]: Oh, I think I'm supposed to used Date.current instead of Date.today maybe.

Comment (nickcianci) [reply spec/models/adoption_app_spec.rb]: https://stackoverflow.com/questions/12748626/when-should-datetime-now-utc-vs-time-current-utc-be-used-in-rails

Comment (markottaviani) [reply spec/models/adoption_app_spec.rb]: Yeah I think that's it
```
Date.current is 2020-01-29
Date.today is 2020-01-30
```

Comment (markottaviani) [reply spec/models/adoption_app_spec.rb]: Fix pushed

Comment (TheDudeWithTheThing) [reply spec/models/adoption_app_spec.rb]: Timecop is great for this stuff too

**Pull request merged**